packages/core/core/src/Parcel.js

```diff
@@ -110,7 +110,7 @@ export default class Parcel {
     await resolvedOptions.cache.ensure();
 
     let {dispose: disposeOptions, ref: optionsRef} =
-      await this.#farm.createSharedReference(resolvedOptions);
+      await this.#farm.createSharedReference(resolvedOptions, false);
```

Comment:
Serializing the options aren't cacheable because of the MemoryFS which has a side effect in the `serialize()` method... 😬